.vnurc

```diff
@@ -4,7 +4,7 @@
 # Proposed, tracking in gh-429
 Bad value “” for attribute “aria-activedescendant” on element “ul”:.*
 # Ignoring for Slider examples because it is forcing rule with role=img
-Element “img” is missing required attribute “alt”.
+.*“img”.*“alt”.*
```

Comment:
Might be good to just list out the new filter patterns so that any new ones are explicitly suppressed